### PR TITLE
new conda-forge python feedstock recipes for google-cloud-container, grpcio-gcp and json-merge-patch

### PR DESCRIPTION
**Pull request by BrentDorsey**
This is my first conda-forge commit.

This pull request includes 3 new conda-forge python feedstock recipes:

- [google-cloud-container](https://github.com/googleapis/google-cloud-python/tree/master/container)
- [grpcio-gcp](https://github.com/GoogleCloudPlatform/grpc-gcp-python)
- [json-merge-patch](https://github.com/OpenDataServices/json-merge-patch) : apache-airflow 1.10 install requirement defined in setup.py [here](https://github.com/apache/airflow/blob/v1-10-stable/setup.py#L309)

[google-cloud-container](https://github.com/googleapis/google-cloud-python/tree/master/container) and [grpcio-gcp](https://github.com/GoogleCloudPlatform/grpc-gcp-python) are required to build apache-airflow 1.10 with the [gcp_api](https://github.com/apache/airflow/blob/v1-10-stable/setup.py#L174) extra.

Thanks,

Brent Dorsey
Hill Country Machine Learning

### CODE REVIEW

File: recipes/google-cloud-container-feedstock/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --verbose"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - google-api-core>=1.6.0,<2.0.0dev
```
Comment (chrisburr) [inline recipes/google-cloud-container-feedstock/meta.yaml]: There needs to be a space here between the package name and the version constraint:
```suggestion
    - google-api-core >=1.6.0,<2.0.0dev
```
(and the same for the others)

File: recipes/google-cloud-container-feedstock/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    - pytz
+    - futures>=3.0.5
+
+test:
+  imports:
+    - google.cloud
+    - google.cloud.container_v1
+    - google.cloud.container_v1.proto
+  requires:
+    - coverage
```
Comment (chrisburr) [inline recipes/google-cloud-container-feedstock/meta.yaml]: Ate these needed for the import tests?

File: recipes/grpcio-gcp-feedstock/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --verbose"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - grpcio>=0.2.2  # must equal grpcio-gcp version
```
Comment (chrisburr) [inline recipes/grpcio-gcp-feedstock/meta.yaml]: How about this
```suggestion
    - grpcio >={{ version }}  # must equal grpcio-gcp version
```
or do you mean:
```suggestion
    - grpcio =={{ version }}  # must equal grpcio-gcp version
```

File: recipes/google-cloud-container-feedstock/meta.yaml (hunk 1 above)
Comment (BrentDorsey) [reply recipes/google-cloud-container-feedstock/meta.yaml]: Space inserted between package and version in all requirement sections in all recipe meta.yaml's.

File: recipes/google-cloud-container-feedstock/meta.yaml (hunk 2 above)
Comment (BrentDorsey) [reply recipes/google-cloud-container-feedstock/meta.yaml]: python packages removed from test.requires

File: recipes/grpcio-gcp-feedstock/meta.yaml (hunk 3 above)
Comment (BrentDorsey) [reply recipes/grpcio-gcp-feedstock/meta.yaml]: `grpcio >={{ version }}` is what I meant, requirement comes from [grpc-gcp-python/src/setup.py](https://github.com/GoogleCloudPlatform/grpc-gcp-python/blob/5a2cd9807bbaf1b85402a2a364775e5b65853df6/src/setup.py#L32).  

grpcio requirement version refactored to use Jinja templating.

**Comment by BrentDorsey** [comment]
Hello Chris,

Thanks for the lightening fast feedback!  Pull request has been updated, all the changes you suggested have been implemented.

I also inserted `*.iml` into .gitignore to exclude Intellij IDEA project module definition files.

Thanks,

Brent Dorsey

**Pull request merged**